slang.h

```diff
@@ -3378,12 +3394,31 @@ namespace slang
     };
 }
 
+// Passed into functions to create globalSession to identify the API version client code is
+// using. 
 #define SLANG_API_VERSION 0
 
+/* Create a global session, with built in StdLib.
+
+@param apiVersion Pass in SLANG_API_VERSION
+@param outGlobalSession (out)The created global session. 
+*/
 SLANG_API SlangResult slang_createGlobalSession(
     SlangInt                apiVersion,
     slang::IGlobalSession** outGlobalSession);
 
+/* Create a global session, but do not set up the stdlib. The stdlib can
+then be loaded via loadStdLib or compileStdLib
+
+@param apiVersion Pass in SLANG_API_VERSION
+@param outGlobalSession (out)The created global session that doesn't have a StdLib setup.
+
+NOTE! API is experimental and not ready for production code 
+*/
+SLANG_API SlangResult slang_createGlobalSessionWithoutStdLib(
+    SlangInt                apiVersion,
+    slang::IGlobalSession** outGlobalSession);
+
```

Comment:
It is probably fine to have this operation for now, but this points to the fact that we *probably* need to start migrating to a model where `slang_createGlobalSession` or whatever we use takes a flexible/extensible "descriptor" of some kind (whether it is an array of key-value pairs, or just a `struct` that embeds a size or version field).